test/OpenTelemetry.Instrumentation.AspNetCore.Tests/RouteTests/RoutingTestCases.json

```diff
@@ -42,7 +46,8 @@
     "path": "/SomePath/SomeString/2",
     "expectedStatusCode": 200,
     "currentHttpRoute": null,
-    "expectedHttpRoute": "SomePath/{id}/{num:int}"
+    "currentMetricRoute": "SomePath/{id}/{num:int}",
+    "expectedHttpRoute": "SomePath/{id}/{num}"
```

Comment:
The route constraint is being lost

Reply:
This is intentional change:

pros:
- produces similar routes as asp.net mvc5
- reduces cardinality
- simplifies the template

negs:
- no visibility into same routes but different constraints (possibly very rare case)